.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
Can avoid pushing this file to github by having a .gitignore file that helps in avoiding adding things related to github